xtext-website/_posts/releasenotes/2022-12-31-version-2-29-0.md

```diff
@@ -28,7 +28,31 @@ Xtext/Xtend now support Java 17 as source and target too. This requires newer Ec
 
 ## Changes / Enhancements
 
-TODO Incremental Standlone Builder
+The Incremental Standlone Builder is now capable of performing incremental builds. Depending on the changes that happened since the previous compile run, only a subset of the model files need to be processed. Both model and Java file changes as well as class-path changes are taken into account. The feature is opt-in since it might break backwards compatibility for DSLs that use non-default code generation patterns. Incremental builds can also be enabled for the Xtext maven plugin. [https://github.com/eclipse/xtext-extras/issues/805](xtext-extras#805).
+
+```xml
+<plugins>
```

Comment:
https://github.com/eclipse/xtext/blob/website-master/xtext-website/documentation/350_continuous_integration.md
should this be applied here too

Reply:
Yes, done.